README.md

```diff
@@ -52,7 +52,7 @@ We also use a [custom style](https://docs.errata.ai/vale/styles) to lint our doc
 
 ### Spelling Errors
 
-If Vale detects a spelling error, it will output it as an error to the console. If you want to ignore a spelling error, you can add the word to the [spelling-execeptions](./vale/styles/Flipt/spelling-exceptions.txt) file in our custom style.
+If Vale detects a spelling error, it will output it as an error to the console. If you want to ignore a spelling error, you can add the word to the spelling-execeptions (`.vale/styles/Flipt/spelling-exceptions.txt`) file in our custom style.
```

Comment:
this was triggering a false positive for the broken links check above, so just made it a non-link